tests/integration/cors.feature

```diff
@@ -1,30 +1,30 @@
 Feature: CORS headers and preflight requests
-
+    
     Scenario: Preflight config request
         Given that I want to make an OPTIONS request
         And that the request "Origin" header is "http://local.dev"
         And that the request "Access-Control-Request-Method" header is "POST"
         And that the request "Access-Control-Request-Headers" header is "Authorization, Content-Type, Accept"
         When I request "/posts"
         Then the guzzle status code should be 200
-        And the "Access-Control-Allow-Origin" header should be "*"
+        And the "Access-Control-Allow-Origin" header should be "http://local.dev"
```

Comment:
The new version of the cors lib enforces returning the origin instead of *, this is correct according to the w3c spec but so is returning *

Reply:
How do we set what origins are allowed? ie. for .io we need to allow `*.ushahidi.io` for the client to work right?

Reply:
It works the same way as before but in the previous version it would return * as the allowed origin where as in this version it returns the request origin - provided it passes the test. We set the origins via the config file, that has remained the same thankfully.

Reply:
Ah ok. That makes sense.
So we would configure an origin like *.ushahidi.io, then a request from qa.ushahidi.io would pass the test and the response would return qa.ushahidi.io?